docs/developer/contributing/development-documentation.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[development-documentation]]
+=== Documentation during development
+
+Docs should be written during development and accompany PRs when relevant. There are multiple types of documentation, and different places to add each.
+
+[float]
+==== Developer services documentation
+
+Documentation about specific services a plugin offers should be encapsulated in:
+
+* README.asciidoc at the base of the plugin folder.
+* Typescript comments for all public services.
+
+[float]
+==== End user documentation
+
+Documentation about user facing features should be written in http://asciidoc.org/[asciidoc] at
+{kib-repo}/tree/master/docs[https://github.com/elastic/kibana/tree/master/docs]
+
+To build the docs, you must clone the https://github.com/elastic/docs[elastic/docs]
+repo as a sibling of your {kib} repo. Follow the instructions in that project's
+README for getting the docs tooling set up.
+
+**To build the docs and open them in your browser:**
```

Comment:
Can you specify that `node scripts/docs.js --open` builds the Kibana docs? There are other product devs that tried to use this command on their own docs, and it didn't work.

Reply:
So just leave out the part about this command opening the docs up in the browser?